src/Introduction.md

```
@@ -10,33 +10,33 @@ In this book we are going to write and learn about operating systems together!
-> *Note*: ALL the syntax highlighting of the Rust code is custom and create by me! If you see and bug, please write in the comments or submit an [issue](https://github.com/sagi21805/mdbook-rust-highlight).
+> *Note*: ALL the syntax highlighting of the Rust code is custom and create by me! If you see any bugs, please write in the comments or submit an [issue](https://github.com/sagi21805/mdbook-rust-highlight).
 
 ## Base Knowledge
 
-This book will be technical, and will assume a little bit of a programming knowledge background, but not necessarily in rust
+This book will be technical, and will assume a little bit of a programming knowledge background, but not necessarily in Rust.
 
 If you are not coming from a low level programming knowledge that's fine!
 
-Just make sure you know this stuff or learn it as you read. Also if in any place on this book I take some things for granted, please, open an issue [here](https://github.com/sagi21805/LearnixOS-Book) and let me know so I could explain it better.
+Just make sure you know this stuff or learn it as you read. Also, if in any place of this book I take some things for granted, please, open an issue [here](https://github.com/sagi21805/LearnixOS-Book) and let me know so I can explain it better.
```

markdownlint-cli2 (0.18.1)

Link text should be descriptive

(MD059, descriptive-link-text)